src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -293,12 +293,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
                 $domainMessagesCount = \count($list);
 
-                if ($sort = $input->getOption('sort')) {
-                    $sort = strtolower($sort);
-                    if (!\in_array($sort, self::SORT_ORDERS, true)) {
-                        $errorIo->error(['Wrong sort order', 'Supported formats are: '.implode(', ', self::SORT_ORDERS).'.']);
-
-                        return 1;
+                if ($input->hasOption('sort')) {
+                    if ($sort = $input->getOption('sort')) {
```

Comment:
Can this ever be false?

Reply:
getOption() returns null if no value has been defined.

Line 296: if the `sort` option has been defined, even if it has no value.
Lines 297-304: if a value has been defined, check that it's valid
Line 307: if the option has been explicitly set to 'desc', makes an rsort()
Line 398: we do an asort() in all other cases

Reply:
Why not defining the default value on the option definition instead of dealing with null and only allow asc and desc as strings and no empty value?

Reply:
I'm ok with this technicaly, but it would change the default behavior, where `--sort=asc` would be applied even if the `--sort` option is not specified. I didn't get into this because the goal is just to fix a bug, and I don't know if the default order responds to any particular logic.

Reply:
I would also set `ASC` as the default value of the option instead of allowing `null` which does not mean anything.